lib/LedgerSMB/Template/HTML.pm

```diff
@@ -34,6 +34,8 @@ sub escape {
     return undef unless defined $vars;
     #$vars = encode_entities($vars);
     $vars = escape_html($vars);
+    #Allow back only a few decorations.
+    $vars =~ s~&lt;(/?[biu]|br)&gt;~<$1>~g;
```

Comment:
If we only aim for tooltips, I’d rather do that in https://github.com/ylavoie/LedgerSMB/blob/5e00f2c217c3288a6cd7a19cc32c3ddd92de8772/UI/lib/elements.html#L439, otherwise every template will have to look for all elements, searching for tooltips that may need unescaping.